Dockerfile

```diff
@@ -0,0 +1,68 @@
+# Pull from the latest Ruby version
+# We don't use slim or alpine variants as we invariably need build tooling, which
+# the base image includes for us.
+FROM ruby
+
+# Avoid issues with file encoding in Ruby by setting these two environment variables
+# This tells the underlying OS to expect UTF-8 encoding (e.g. UTF-8 encoding in the US language)
+ENV LANG "C"
+ENV LC_ALL C.UTF-8
+
+
+# Curl is installed to make it possible to set up PPAs below - it is
+# removed further down.
+RUN apt-get update -qq &&\
+    apt-get install -y curl 
+
+# Install Node.js PPA for asset management
+# As of writing, Node 10 is the most recent LTS.
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash
+
+# Install Google Chrome PPA. Chrome is required for headless system tests.
+RUN curl -q https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+
+# Install system dependencies, and remove curl now that we have PPAs set up
+# We also clean out system files we don't need to reduce image size:
+#   * /usr/share/man - manual pages
+#   * /usr/share/locales - we don't need to support multiple languages at the OS level
+#   * /var/cache/apt/arhives - we don't need to hold onto deb packages once they're installed
+RUN apt-get update -qq &&\
+    apt-get upgrade -y &&\
+    apt-get install -y google-chrome-stable libpq-dev nodejs --no-install-recommends &&\
+    apt-get purge -y curl &&\
```

Comment:
🐘 You might be able to purge `apt-transport-https` if its installed, as that's only needed for getting PPAs as well iirc.

Reply:
@G-Rath unless the bash pipes are adding apt-transport-https, I'm not adding it. 
It does raise an interesting point about what other packages might be included in the default image though

Reply:
Totally fair - I only mention it b/c YVW has a dockerfile that tries to remove as much as possible, and I remember that adding the PPAs for Node failed if `apt-transport-https` wasn't installed.

That tracks given that they're in `https`, so I suspect it's already installed via the default image.

Reply:
@G-Rath `apt list` shows `apt-transport-https` not installed, so I assume that it's good